ext/build_defaults.yaml

```diff
@@ -14,6 +14,7 @@ yum_host: 'yum.puppetlabs.com'
 yum_repo_path: '/opt/repository/yum/'
 build_gem: TRUE
 build_dmg: TRUE
+msi_name: 'puppet-agent'
 build_msi:
   puppet_for_the_win:
     ref: '2476cd007b1b52fb1b9f22202b56218629626b5c'
```

Comment:
This ref needs to be updated to latest for this to not fail, just fyi, And I believe we want to wait until https://github.com/puppetlabs/puppet_for_the_win/pull/84 is merged for that.